src/main/resources/templates/index.html

```diff
@@ -17,6 +17,26 @@ <h1>Welcome to Neighborhood Assurance!</h1>
 		<input type="text" name="state" placeholder="Test: Search for OH"/>
 		<button name="submit" type="submit" value="0">Submit</button>
 	</form>
+	<br/>
+	<br/>
+Search is not working yet, please use this button
+	<br/>
+	<br/>
+	<a href="resultSearch">
+			<button>Search</button>
+	</a>
+	
+	
+	</br>
+	</br>
+Login Interface is not working yet </br></br>

```

Comment:
Adding code that does not work/is not fully implemented is adding technical debt.